docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-installer.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Upgrade YugabyteDB Anywhere using YBA Installer
+headerTitle: Upgrade YugabyteDB Anywhere
+linkTitle: Upgrade installation
+description: Use YBA Installer to upgrade YugabyteDB Anywhere
+menu:
+  preview_yugabyte-platform:
+    identifier: upgrade-yp-1-installer
+    parent: upgrade
+    weight: 80
+type: docs
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+
+  <li>
+    <a href="../upgrade-yp-installer/" class="nav-link active">
+      <i class="fa-solid fa-building"></i>YBA Installer</a>
+  </li>
+
+  <li>
+    <a href="../upgrade-yp-replicated/" class="nav-link">
+      <i class="fa-solid fa-cloud"></i>Replicated</a>
+  </li>
+
+  <li>
+    <a href="../upgrade-yp-kubernetes/" class="nav-link">
+      <i class="fa-regular fa-dharmachakra" aria-hidden="true"></i>Kubernetes</a>
+  </li>
+
+</ul>
+
+To upgrade using YBA Installer, first download the version of YBA Installer corresponding to the version of YBA you want to upgrade to.
```

Comment:
Maybe call out you can only upgrade to a higher version and downgrading is NOT supported, so this is a one way action.